src/utils/tokenHelper.js

```diff
@@ -41,9 +41,32 @@ function getTokenBySymbol(symbol) {
   return tokensBySymbols[symbol] || { symbol };
 }
 
+const isSymbolInTokenWhitelist = symbol => {
+  return Boolean(
```

Comment:
Let's create a set which includes the symbols. The set should be initialized once when the application start

Reply:
@aminlatifi done